### ISSUE TITLE
508-defect-2: screenreaders and users of speech to text software, design system Back to Top component hides text on narrow screens

### ISSUE THREAD

**Issue by AngelaFowler82** [department-of-veterans-affairs/va.gov-team#58413]
### Point of contact

Angela Fowler

### Severity level

2, Serious. Should be fixed in 1-2 sprints post-launch.

### Details

...
On a narrow screen, the back to top component displays visually as an arrow presumably pointing to the top of the page. The text label is hidden and there is no accessible label. This causes screen readers to announce the button as "unlabeled 0." It would also make it difficult for users of access technologies such as Dragon Naturally Speaking to use the component as they would not know how to refer to it.

### Reproduction steps

1.
2.
3.
...

### Proposed solution or next steps

Do not hide the text label on narrow screens.

### References, articles, or WCAG support

1.
2.
3.
...

### Type of issue

- [ ] Screenreader
- [ ] Keyboard
- [ ] Focus
- [ ] Headings
- [ ] Color
- [ ] Zoom
- [ ] Semantics
- [ ] Axe-core
- [ ] Something else

**Comment by jamigibbs** [comment]
I reopened this so that until we're able to get confirmation from an accessibility review for the development and design updates we made: https://github.com/department-of-veterans-affairs/va.gov-team/issues/60812#issuecomment-1644091586
